client/src/views/admin/index.vue

```diff
@@ -26,9 +26,18 @@
                 </b-container>
             </b-card>
         </b-container>
+      </b-card>
+    </b-container>
+    <router-link to="/admin/userList">User</router-link>
+    <br/>
+    <router-link to="/admin/list">List Admins</router-link>
+    <br/>
+    <router-link to="/admin/register">Register Admin</router-link>
+    <router-view></router-view>
+  </div>
         <router-link to="/admin/userList">User</router-link>
         <router-view></router-view>
-    </div>
```

Comment:
wrong way of writing component
similar to react components, vue components must also be enclosed in single div
- check code in master branch for this file and follow accordingly.

Please make sure to run client using `npm run serve` and test to avoid such errors

Reply:
Noted and updated.